regex-syntax/src/ast/mod.rs

```diff
@@ -307,6 +310,9 @@ impl fmt::Display for ErrorKind {
                 write!(f, "invalid repetition count range, \
                            the start must be <= the end")
             }
+            RepetitionQuantifierDecimalMissing => {
+                write!(f, "Repetition quantifier expects a valid decimal")
```

Comment:
Could this match the style of surrounding style of error messages? i.e., Start with a lowercase letter.